### ISSUE TITLE
Wait forever, if 'nc' not installed ;)

### ISSUE THREAD

**Issue by jedie** [andreiavrammsd/wait-for-it#2]
The script doesn't check if 'nc' is installed... 
If it's not installed, then it will wait and wait and wait ... ;)

Don't know if this is a good idea:
```
check=$(nc &> /dev/null; echo $?)
if [[ ! "$check" -eq 1 ]]; then
    echo Error: 'nc' not installed?!?
    exit -1
fi
```

exit code from `nc` is 1 if it's installed... If it's not available: 127

**Comment by andreiavrammsd** [comment]
I know, I just wrote the whole thing only with nc in mind, for a use case I had. I should've at least mention that dependency. Thanks for the tip!

If you want to create a pull request, I'd be happy to merge it.

**Comment by andreiavrammsd** [comment]
Fixed in f326d7070b84b037809ff462f2dc1c2e150972e6

Thanks!
